apps/web/public/static/locales/en/common.json

```diff
@@ -1065,6 +1065,8 @@
   "warning_instant_meeting_experimental": "Experimental: Instant Meeting Events are currently experimental.",
   "starting": "Starting",
   "disable_guests": "Disable Guests",
+  "disable_autofill_on_booking_page": "Disable autofill on booking pages",
+  "disable_autofill_on_booking_page_description": "Prevent booking forms from being automatically filled with URL parameters for all users in this organization",
```

Comment:
Accordingly update this description to mention that both URL params and session based prefill is disabled.